src/coreclr/vm/assembly.cpp

```diff
@@ -686,6 +686,21 @@ Assembly *Assembly::CreateDynamic(AppDomain *pDomain, ICLRPrivBinder* pBinderCon
             pAssem->SetIsTenured();
             pRetVal = pAssem;
         }
+
+        {
```

Comment:
This block is not needed, can you please remove the curly braces?

Reply:
Block removed